CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,114 @@
+This document was copied from its home at https://publiclab.org/conduct on October 12, 2017. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document.
```

Comment:
@jywarren do you think we need to change the date?

Reply:
Actually, the doc requires many tweaks so I have opened a issue at https://github.com/publiclab/plots2/issues/4730. Thanks!